ui/prometheus-plugin/package.json

```diff
@@ -12,13 +12,16 @@
     "lint:fix": "eslint --fix src --ext .ts,.tsx"
   },
   "dependencies": {
-    "@perses-ui/core": "^1.0.0",
+    "@perses-dev/core": "^0.1.0",
+    "@perses-dev/plugin-system": "^0.1.0",
     "date-fns": "^2.28.0",
     "lezer": "^0.13.5",
     "lezer-promql": "^0.20.0",
-    "react": "^17.0.2",
     "react-query": "^3.34.12"
   },
+  "peerDependencies": {
+    "react": "^17.0.2"
```

Comment:
I guess you should add it as a `devDependency` as well

Reply:
Yup, good catch.